plugins/chain_api_plugin/chain_api_plugin.cpp

```diff
@@ -703,75 +704,71 @@ static float64_t to_softfloat64( double d ) {
    return *reinterpret_cast<float64_t*>(&d);
 }
 
-fc::variant get_global_row( const chain::database& db, const chain::abi_def& abi, const chain::abi_serializer& abis, const fc::microseconds& , bool ) {
-//   const auto table_type = get_index_type(abi, N(global));
-//   EOS_ASSERT(table_type == read_only::KEYi64, chain::contract_table_query_exception, "Invalid table type ${type} for table global", ("type",table_type));
+get_producers_result chain_api_plugin_impl::get_producers( const get_producers_params& p ) const {
+    auto& chaindb = chain_controller_.chaindb();
 
-// TODO: Removed by CyberWay
-//   const auto* const table_id = db_controller_.find<chain::table_id_object, chain::by_code_scope_table>(boost::make_tuple(config::system_account_name, config::system_account_name, N(global)));
-//   EOS_ASSERT(table_id, chain::contract_table_query_exception, "Missing table global");
-//
-//   const auto& kv_index = db_controller_.get_index<key_value_index, by_scope_primary>();
-//   const auto it = kv_index.find(boost::make_tuple(table_id->id, N(global)));
-//   EOS_ASSERT(it != kv_index.end(), chain::contract_table_query_exception, "Missing row in table global");
-//
-//   vector<char> data;
-//   read_only::copy_inline_row(*it, data);
-//   return abis.binary_to_variant(abis.get_table_type(N(global)), data, abi_serializer_max_time_ms, shorten_abi_errors );
-   return fc::variant();
+    const cyberway::chaindb::index_request request{N(cyber.govern), N(cyber.govern), N(producer), cyberway::chaindb::names::primary_index};
+
+    auto producers_it = chaindb.begin(request);
+    cyberway::chaindb::cursor_request cursor_req{N(cyber.govern), producers_it.cursor};
+    std::vector<fc::variant> rows;
+    std::string next_producer;
+    uint64_t total_votes = 0;
+
+    for (size_t count = 0; producers_it.pk != cyberway::chaindb::end_primary_key; producers_it.pk = chaindb.next(cursor_req)) {
+
+        const auto value = chaindb.value_at_cursor(cursor_req);
+        const auto account = value["account"].as_string();
+        const auto votes = value["votes"].as_uint64();
```

Comment:
votes field is int64 and in the current implementation it's negative (-1) for not elected producers